spec/factories/miq_template.rb

```diff
@@ -1,4 +1,7 @@
 FactoryGirl.define do
   factory :miq_template do
+    name "ubuntu-16.04-stable"
+    location "Minneapolis, MN"
+    vendor "openstack"
```

Comment:
> It is nice to keep templates at the bare minimum.

Exactly, which is why these required attributes for this object should be here :) You can't create one without them.

Sidenote: We need to stop using *just* factories all over and start utilizing useful traits. (ie instead of `:template_with_location`, `:template, :with_location`)